### ISSUE TITLE
libfmod not found when I run cpp-tests

### ISSUE THREAD

**Issue by MatthieuG9** [cocos2d/cocos2d-x#16511]
When I run cpp-tests I got the error `error while loading shared libraries: libfmod.so.6: cannot open shared object file: No such file or directory`

Configuration :
- cocos2d-x version: 3.13
- devices test on: Linux Ubuntu 14.04 i686

Steps to Reproduce:
1. Clone the cocos repository,
2. python download-deps.py
2. git submodule update --init
2. ./install-deps-linux.sh
2. Build tests files
3. Run cpp-tests

To fix this error, I copied libfmod and lifmodL into `/usr/local/lib` folder and updated lib cache with `ldconfig`.

Suggestion : download-deps.py or install-deps-linux.sh should automatically install libfmod ?

**Comment by JOOpdenhoevel** [comment]
Yes, I know this problem and it will be fixed when #16410 is merged. Basically, fmod is going to be replaced by openal.

**Comment by ricardoquesada** [comment]
yes, as @Janonard  said.
In the meantime, you have to install fmod manually.

**Comment by genaside** [comment]
So what is the status? can I go ahead and use #16410?

**Comment by cirosantilli** [comment]
Possible duplicate of: https://github.com/cocos2d/cocos2d-x/issues/15137
